dsc_lib/src/configure/mod.rs

```diff
@@ -150,6 +152,40 @@ impl Configurator {
         Ok(result)
     }
 
+    pub fn invoke_export(&self, _error_action: ErrorAction, _progress_callback: impl Fn() + 'static) -> Result<ConfigurationExportResult, DscError> {
+        let (config, messages, had_errors) = self.validate_config()?;
+        let mut result = ConfigurationExportResult::new();
```

Comment:
You don't need to use the default constructor here, instead, use this syntax:

```rust
let mut result = ConfigurationExportResult {
  messages,
  had_errors
};
```

Reply:
Updated.